x-pack/legacy/plugins/security/public/views/management/edit_role/index.js

```diff
@@ -87,9 +85,12 @@ const routeDefinition = (action) => ({
       }
       return [];
     },
-    privileges() {
+    kibanaPrivileges() {
       return kfetch({ method: 'get', pathname: '/api/security/privileges', query: { includeActions: true } });
     },
+    builtinESPrivileges() {
```

Comment:
Once we de-angularize, I think it'd make sense to make the `ElasticsearchPrivileges` component (or similar) responsible for this, rather than passing it down from the `EditRolesPage` component. I opted for this approach to stay consistent with the rest of the page, and we can come up with a holistic data-access approach for the entire screen once we remove angular.